src/mscorlib/src/System.Private.CoreLib.txt

```diff
@@ -854,6 +854,7 @@ BadImageFormat_StreamPositionInvalid = Corrupt .resources file.  The specified p
 BadImageFormat_ResourcesDataInvalidOffset = Corrupt .resources file. Invalid offset '{0}' into data section.
 BadImageFormat_NegativeStringLength = Corrupt .resources file. String length must be non-negative.
 BadImageFormat_ParameterSignatureMismatch = The parameters and the signature of the method don't match.
+BadImageFormat_BadILFormat = Bad IL format.
```

Comment:
Are you referring to "Arg_BadImageFormatException = Format of the executable (.exe) or library (.dll) is invalid. " I was matching the message that we get for .net core 1.1. Also above message seems confusing that we are loading a dll but in this case it is not. This is a stream.